asciidoc/cypher-workflow.adoc

```diff
@@ -367,13 +400,13 @@ All can be potentially found in results although not all can be used as paramete
 +*+ The null marker is not a type but a placeholder for absence of value.
 For information on how to work with null in Cypher, please refer to <<cypher-manual#cypher-working-with-null, Cypher Manual -> Working with `null`>>.
 
-+**+ Nodes, relationships and paths are passed in results as snapshots of the original graph entities.
++**+ Nodes, Relationships and Paths are passed in results as snapshots of the original graph entities.
```

Comment:
We don't typically capitalise Nodes, Relationships and Paths - was there a reason for this?